src/api/v1/router.ts

```diff
@@ -18,61 +18,124 @@ router.get('/version', (req: Request, res: Response) => {
-    const ethereumAddress = wallet.ethereumAddress
+    const ethereumAddress = wallet.ethereumAddress;
 
     if (email) {
       const encryptedWallet: IEncryptedWallet = await encryptWalletWithEmail(
-          ethereumAddress,
-          wallet.mnemonic,
-          wallet.privateKey,
-          email
-      )
+        ethereumAddress,
+        wallet.mnemonic,
+        wallet.privateKey,
+        email
+      );
 
       const data: IResponseWalletCreateEmail = {
         ...encryptedWallet,
         ethereumAddress,
         email
-      }
+      };
 
       return res.send({ data });
     }
 
     if (phone) {
       const encryptedWallet: IEncryptedWallet = await encryptWalletWithPhone(
-          ethereumAddress,
-          wallet.mnemonic,
-          wallet.privateKey,
-          phone
-      )
+        ethereumAddress,
+        wallet.mnemonic,
+        wallet.privateKey,
+        phone
+      );
 
       const data: IResponseWalletCreatePhone = {
         ...encryptedWallet,
         ethereumAddress,
         phone
-      }
+      };
 
       return res.send({ data });
     }
 
     const error: IResponseError = {
       message: 'No valid identity',
       code: ERROR_CODES.BAD_REQUEST
-    }
+    };
 
     return res.send({ error });
   } catch (err: any) {
-    loggingService.error("Could not create wallet", err.message);
+    loggingService.error('Could not create wallet', err.message);
 
     const error: IResponseError = {
       message: 'Could not create wallet',
       code: ERROR_CODES.INTERNAL_ERROR
-    }
+    };
+
+    return res.status(500).send({ error });
+  }
+});
+
+// CREATE NEW WALLET AND ENCRYPT FOR SERVICE WALLET
+router.post('/service/wallet', async (req: Request, res: Response) => {
+  try {
+    const { email } = req.body;
+    const wallet: IDecryptedWallet = createWallet();
+    const ethereumAddress = wallet.ethereumAddress;
+
+    const encryptedWallet: IEncryptedWallet = await encryptWalletForService({
+      ethereumAddress,
+      mnemonic: wallet.mnemonic,
+      privateKey: wallet.privateKey,
+      email
+    });
+
+    const data: IResponseWalletCreateEmail = {
+      ...encryptedWallet,
+      ethereumAddress,
+      email
+    };
+
+    return res.send({ data });
+  } catch (err: any) {
+    loggingService.error('Could not create wallet for service', err.message);
+
+    const error: IResponseError = {
+      message: 'Could not create wallet for service',
+      code: ERROR_CODES.INTERNAL_ERROR
+    };
 
-    return res.status(500).send({ error })
+    return res.status(500).send({ error });
   }
 });
 
+// ENCRYPT IMPORTED WALLET FOR SERVICE WALLET
+router.post('/service/wallet/import', async (req: Request, res: Response) => {
+  try {
+    const { email, ethereumAddress, mnemonic, privateKey } = req.body;
+
+    const encryptedWallet: IEncryptedWallet = await encryptWalletForService({
+      ethereumAddress,
+      mnemonic,
+      privateKey,
+      email
+    });
```

Comment:
To prevent issues that could arise from passing in a private key that doesn't match the mnemonic, we should probably just accept the `mnemonic` and derive the ethereum address & private key from that mnemonic. 

Also, same comment here as above - we should use an account ID instead of an email address for service accounts.

Reply:
ahhhh ok
then maybe we should support passing in EITHER a private key OR a mnemonic - but passing in both prevents us from guaranteeing consistency